src/Core/Lock.php

```diff
@@ -0,0 +1,142 @@
+<?php
+/**
+ * @file src/Util/Lock.php
```

Comment:
This tag is wrong and should be deleted. Additionally, all references to `Util\Lock` have to be updated through the project.